vane-core/src/main/java/org/oddlama/vane/core/data/DurabilityOverrideData.java

```diff
@@ -188,7 +188,7 @@ protected boolean is_durability_lore(final Component component) {
 	private int damage_clamp(final int fake_damage, final int fake_max, final float dangerous_interpolated_damage, final short max) {
 		// This can only happen in weird edgecases, in which case we recover by breaking the item the next time
 		// the player uses it. Much better then showing negative values in the fake durability lore.
-		if (fake_damage >= fake_max) return max - 1;
+		if (fake_damage > fake_max) return max - 1;
```

Comment:
I'm not sure whether this change breaks something. This is one of the few things that weren't written by me, so we have to find out whether that change is fine.

I just saw that before, `max-1` would be returned if `fake_damage == fake_max`, but now we get `max`. Essentially, the if statement below this one was dead code before.

Reply:
Oh wait, nevermind. It seems like I already rewrote this code and the whole `DurabilityOverrideData.java` class isn't used (the code is now all in DurabilityManager.java). I just forgot to remove it when I did that :D

So instead, please just delete the whole file! 😄